compressor/conf.py

```diff
@@ -70,6 +70,12 @@ class CompressorConf(AppConf):
     OFFLINE_MANIFEST = 'manifest.json'
     # The Context to be used when TemplateFilter is used
     TEMPLATE_FILTER_CONTEXT = {}
+    # URL-less mode: you do offline compression & decompression regardless of settings.STATIC_URL
+    # and settings.COMPRESS_URL so your assets become re-locatable for any URLs.
+    # Affects manifest file contents and manifest key determination, but not the assets contents.
+    OFFLINE_URLLESS = False
```

Comment:
i'd say in this case `URL_PLACEHOLDER` can still be a setting because that's a good place for global constants (or is there a good alternative?), but not document it because people should generally not care about it.